docs/multicluster/antctl.md

```diff
@@ -36,57 +36,96 @@ To see the usage examples of these commands, you may also run `antctl mc get [su
 
 ## antctl mc create
 
-`antctl mc create` command can create access-token and other resources like ClusterSet, ClusterClaims for
-Antrea Multi-cluster setup
-
-+ `antctl mc create accesstoken` command can create accesstoken for member clusters.
-+ `antctl mc create clusterclaims` command can create two ClusterClaims in a leader or member cluster. One for the leader or member cluster, and another for the ClusterSet.
-+ `antctl mc create clusterset` command can create a ClusterSet in a leader or member cluster.
+`antctl mc create` command can create tokens for member clusters to join a ClusterSet. The command will
+also create a Secret to store the token, as well as a ServiceAccount and a RoleBinding. The `--output-file`
+option saves the member token Secret manifest to a file.
 
 ```bash
-anctcl mc create accesstoken [NAME] [-n NAMESPACE] [--serviceaccount SERVICE_ACCOUNT] [--role-binding ROLE_BINDING]
-antctl mc create clusterclaims [-n NAMESPACE] [--clusterset-id CLUSTERSET_ID] [--cluster-id CLUSTER_ID]
-antctl mc create clusterset [NAME] [-n NAMESPACE] [--leader-server LEADER_SERVER] [--service-account SERVICE_ACCOUNT] [--secret SECRET] [--leader-cluster LEADER_CLUSTER_ID]
+anctcl mc create membertoken [NAME] [-n NAMESPACE] [--output-file OUTPUT_FILE]
 ```
 
 To see the usage examples of these commands, you may also run `antctl mc create [subcommand] --help`.
 
-## antctl mc add
+## antctl mc deploy
+
+`antctl mc deploy` command deploys Antrea Multi-cluster Controller to a leader or member cluster.
 
-`antctl mc add` command can add a new member cluster to a ClusterSet.
++ `antctl mc deploy leadercluster` command deploys Antrea Multi-cluster Controller to a leader cluster and imports
+  all the Antrea Multi-cluster CRDs.
++ `antctl mc deploy membercluster` command deploys Antrea Multi-cluster Controller to a member cluster and imports
+  all the Antrea Multi-cluster CRDs.
 
 ```bash
-antctl mc add membercluster [CLUSTER_ID] [-n NAMESPACE] [--clusterset CLUSTERSET] [--service-account SERVICE_ACCOUNT]
+antctl mc deploy leadercluster [--antrea-version ANTREA_VERSION] [-n NAMESPACE] [-f PATH_TO_MANIFEST]
```

Comment:
Should we use:

antctl mc deploy leadercluster -n NAMESPACE [--antrea-version ANTREA_VERSION] [-f PATH_TO_MANIFEST]

assuming Namespace is mandatory?

Reply:
Done.